frontend/src/utils/getImages.js

```diff
@@ -1,8 +1,8 @@
 import { LIMIT } from "../static/constants";
 
-const axios = require("axios");
+import axios from "axios";
 const hostName = `${
-  process?.env?.REACT_APP_API_URL || "http://localhost:5000"
+  import.meta?.env?.VITE_APP_API_URL || "http://localhost:5000"
```

Comment:
hmm this changes things will also break the production. I will need to adjust the environment variables.